DigitalLearningSolutions.Web/Views/LearningMenu/Section/_ConsolidationExerciseCard.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
+@model SectionContentViewModel
+
+<div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
+  <div class="nhsuk-card__content">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <h3 class="nhsuk-card__heading" id="consolidation-exercise-name">
+          Consolidation Exercise
+        </h3>
+      </div>
+    </div>
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <a class="nhsuk-button nhsuk-button--secondary nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0"
```

Comment:
Any reason why this is a secondary button? 🙂

Reply:
As we used a secondary button for the download supporting materials i thought it would be more consistent. I can change this to a regular button if you feel that's more appropriate

Reply:
We used a secondary button in that case because there could be more than one button. In this case this card will only have one button so I think it would be most consistent to make it a primary button 🙂